calculation/calculation_rule.py

```diff
@@ -101,18 +102,10 @@ def calculate(cls, instance, **kwargs):
                     value = float(income) * (rate / 100)
                     return value
             elif context == 'members':
-                cp_params, cd_params = instance.contribution_plan.json_ext, instance.contract_details.json_ext
```

Comment:
the rule has a parameters to say if the family should be included or not, it should not be removed

Reply:
ahh ok, I removed this by accident while resolving a conflict. let me check this again